lib/WP_Auth0_LoginManager.php

```diff
@@ -170,19 +163,11 @@ public function init_auth0() {
       }
     } catch (WP_Auth0_LoginFlowValidationException $e) {
 
-      $msg = __( 'There was a problem with your log in. ', 'wp-auth0' );
-      $msg .= ' '. $e->getMessage();
-      $msg .= '<br/><br/>';
-      $msg .= '<a href="' . wp_login_url() . '">' . __( '← Login', 'wp-auth0' ) . '</a>';
-      wp_die( $msg );
+      $this->die_on_login( $e->getMessage(), $e->getCode() );
```

Comment:
This 2 `die_on_login` calls only differ on the boolean you pass at the end.. This uses the default TRUE, meaning is "LOGIN_LINK" (idk what that means). The other passes FALSE, so its "LOGOUT_LINK" (again idk). How is that an exception type on the same "try logic" could tell you that?

Reply:
Do you have an internal exception from when the user needs to log in / log out? In that case I think I understand now, and depending on the bool you are adding the link to log in / log out on the message.

Reply:
That whole `try` block needs attention in another PR but, as I understand the intended flow, there are exceptions that are thrown during the login process, `WP_Auth0_LoginFlowValidationException`, and exceptions thrown once the login flow is complete but before you're actually logged in, `WP_Auth0_BeforeLoginException`. That `bool` determines what the messaging is.